api/drizzle/seeder/assign-shops.ts

```diff
@@ -5,26 +5,6 @@ import { products as productSchema } from "../../src/schema/products"
 import { people as peopleSchema } from "../../src/schema/people"
 import { orders as orderSchema } from "../../src/schema/orders"
 
-/**
- * Assign shop_id to products based on their product category.
- */
-export async function assignShopsToProducts() {
-  const products = await db.query.products.findMany({
-    columns: { id: true, categoryId: true },
-    with: { category: true },
-  })
-
-  for (const product of products) {
-    const { shopId } = product.category ?? {}
-
-    await db
-      .update(productSchema)
-      .set({ shopId })
-      .where(eq(productSchema.id, product.id))
-      .returning()
-  }
-}
-
```

Comment:
cc @heypoom i removed it, now images are assigned as the part of products generation